metadata-ingestion/src/datahub/ingestion/source/redshift/redshift.py

```diff
@@ -723,26 +724,14 @@ def gen_dataset_workunits(
             dataset_urn, table, str(datahub_dataset_name)
         )
 
-        dataset_properties = DatasetProperties(
-            name=table.name,
-            created=TimeStamp(time=int(table.created.timestamp() * 1000))
-            if table.created
-            else None,
-            lastModified=TimeStamp(time=int(table.last_altered.timestamp() * 1000))
-            if table.last_altered
-            else TimeStamp(time=int(table.created.timestamp() * 1000))
-            if table.created
-            else None,
-            description=table.comment,
-            qualifiedName=str(datahub_dataset_name),
-        )
```

Comment:
I think we still need these properties... we have `set_description` and `set_display_name`, not sure if we have anything for created, last modified, and qualified name